SAMtags.tex

```diff
@@ -411,17 +411,111 @@ \section{Locally-defined tags}
 
 \begin{appendices}
 \appendix
-\section{SAM Tags History}\label{sec:history}
+\section{Tag History}
 
-This lists the date of each tagged SAM version along with changes that
-have been made while that version was current.  
+This appendix lists when standard tags were initially defined or significantly changed, and other historical events that affect how tags are interpreted or what files they may appear in.
 
-\subsection*{1.5: 23 May 2013 to current}
-\begin{itemize}
-\item Cellular barcode tags CB, CR, and CY added. (May 2018)
-\item Add UMI-related tags (RX, QX, OX, BZ, MI) and clarified usage of sample barcode tag BC. (August 2017)
-\item SAMtags.txt (this file) created with tags from SAMv1 
-\end{itemize}
+\setlength{\parindent}{0pt}
+\newcommand*{\gap}{\vspace*{2ex}}
+
+\subsubsection*{May 2018}
+
+Cellular barcode tags CB, CR, and CY added.
+
+\subsubsection*{November 2017}
+
+CG tag added.
+
+\subsubsection*{August 2017}
+
+Unique molecular identifier tags BZ, MI, OX, QX, and RX added.
+
+Usage of sample barcode tag BC clarified.
+
+\subsubsection*{June 2017}
+
+Corrected the description of the E2 tag, which previously wrongly claimed E2 to be encoded as per {\sf QUAL}.
+
+\subsubsection*{September 2016}
+
+Predefined tags, previously listed as a brief table within the main SAM specification, have been split out into this new document.
+There is now space for clearer and more complete tag descriptions.
+
+\subsubsection*{February 2014}
+
+MC tag added.
+
+\subsubsection*{May 2013}
+
+SAM version number {\tt VN:1.5} introduced, indicating the addition of the {\sc supplementary} flag (bit 0x800).
+
+SA tag added.
+
+\subsubsection*{March 2012}
+
+Descriptions of CT and PT annotation tags significantly clarified.
+
+\subsubsection*{October 2011}
+
+Sample barcode tags QT and RT added, with RT being identified as a deprecated alternative to BC.
+
+% These were actually added in late September as RT/PT, but RT was changed to
+% CT (see samtools-devel, "Potential clash of RT tags (annotation vs barcode)",
+% October 2011) before read-annotation-RT appeared in the wild.
+Read annotation tags CT and PT added.
+
+\subsubsection*{September 2011}
+
+% This was actually August 29th, but let's call it September.
+FZ tag's type changed from {\tt H} to {\tt B,S}-array.
+
+BC and CO tags added.
+
+\subsubsection*{April 2011}
+
+SAM version number {\tt VN:1.4} introduced, indicating the addition of the {\tt B}-array tag type.
```

Comment:
again, I'm not sure why the SAM version should be mentioned in this document.

Reply:
In the case of VN:1.4 and array tags, I would have thought the relevance was obvious — and made obvious by this paragraph. Please (re)read it and suggest how it could be made clearer.